web/main/resources.ejs

```diff
@@ -24,6 +24,15 @@
                         <input type="checkbox" id="defResCheckbox" class="c-switch-input">
                         <span class="c-switch-slider" data-checked="On" data-unchecked="Off"></span>
                     </label>
+
+                    <span class="form-text text-muted ml-2" style="margin-top: calc(0.3rem + 2px)">
+                        Stopped resources:
+                    </span>
+                    &nbsp;&nbsp;
+                    <label class="c-switch c-switch-label c-switch-pill c-switch-success fix-pill-form">
+                        <input type="checkbox" id="stoppedResCheckbox" class="c-switch-input">
+                        <span class="c-switch-slider" data-checked="On" data-unchecked="Off"></span>
```

Comment:
Szia!
Reviewing on behalf of @tabarra. I think updating the labels here to `Show` and `Hide` would clear things up a bit. Otherwise I checked your code and I like the feature, I have a few code related ideas though.

Reply:
Maybe even renaming the switch to `Only stopped resources` or `Show only stopped` would be beneficial.

Reply:
![2](https://user-images.githubusercontent.com/36936216/186997153-426b0442-d5f3-4f6a-8d15-5d2d33c98720.png)
![3](https://user-images.githubusercontent.com/36936216/186997181-0b2c3ab8-b896-4fd2-8b9b-c066c9487dc0.png)

Reply:
Oh well

Reply:
Change the labels then

Reply:
Hali!
I was a little tired last night, but I did it today, I hope it's good.